cognition_objects/pipeline_log.py

```diff
@@ -220,3 +221,63 @@ def update_to_new_diff_structure(
 
     if with_commit:
         general.commit()
+
+
+def get_error_and_time_elapsed_by_conversation_id(
```

Comment:
I'm not sure if we need two methods for basically the same code. 
I think conversation_ids can be Union[Iterable[str],str] and then in code with
```
if not isinstance(value, collections_abc_Iterable):
   conversation_ids = [conversation_ids]

```
or something similar